packages/suite/src/components/suite/modals/AdvancedCoinSettings/components/CustomBlockbookUrls.tsx

```diff
@@ -1,159 +1,344 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
This file is kinda long now

Reply:
https://github.com/trezor/trezor-suite/pull/4779/commits/c1ce85d1c5622a8bff596c4446a7f5bfc00e9f40